serverless/README.md

```diff
@@ -70,8 +70,15 @@ You can configure the library by add the following parameters:
 
 ```yaml
 # Whether to add the Lambda Layers, or expect the user to bring their own. Defaults to true.
+# If this is true, 'pythonLibraryVersion' is required if any Lambda function has a Python runtime, and 'nodeLibraryVersion' is required if any Lambda function has a Node.js runtime.
 addLayers: true
 
+# The Python Lambda Library version to install to the Python Lambda functions. This is required if 'addLayers' is true and you are deploying one or more Python Lambda functions. The latest version is 19.
```

Comment:
Can you add a link to the library releases page on github rather than saying the latest version is 19, which will soon be an inaccurate statement?

Reply:
Will do 👍